awx/ui/client/src/inventories-hosts/inventories/related/groups/list/groups-list.controller.js

```diff
@@ -102,35 +102,51 @@
             $state.go('inventories.edit.groups.edit.nested_groups', {group_id: id});
         };
         $scope.deleteGroup = function(group){
-            $scope.toDelete = {};
-            $scope.strings.deleteModal = {};
-            angular.extend($scope.toDelete, group);
-            if($scope.toDelete.total_groups === 0 && $scope.toDelete.total_hosts === 0) {
-                // This group doesn't have any child groups or hosts - the user is just trying to delete
-                // the group
-                $scope.deleteOption = "delete";
-            }
-            else {
-                $scope.strings.deleteModal.group = InventoryHostsStrings.get('deletegroup.GROUP', $scope.toDelete.total_groups);
-                $scope.strings.deleteModal.host = InventoryHostsStrings.get('deletegroup.HOST', $scope.toDelete.total_hosts);
-
-                if($scope.toDelete.total_groups === 0 || $scope.toDelete.total_hosts === 0) {
-                    if($scope.toDelete.total_groups === 0) {
-                        $scope.strings.deleteModal.deleteGroupsHosts = InventoryHostsStrings.get('deletegroup.DELETE_HOST', $scope.toDelete.total_hosts);
-                        $scope.strings.deleteModal.promoteGroupsHosts = InventoryHostsStrings.get('deletegroup.PROMOTE_HOST', $scope.toDelete.total_hosts);
+            const promises = [];
+
+            Rest.setUrl(group.related.hosts);
+            promises.push(Rest.get());
+
+            Rest.setUrl(group.related.children);
+            promises.push(Rest.get());
+
```

Comment:
I'm assuming that the list `count`s returned by these `group.related` endpoints are equivalent to the old `total_hosts` and `total_groups`. Does this sound right? @mabashian @AlanCoding